R/brmsfit-helpers.R

```diff
@@ -255,7 +255,12 @@ get_cov_matrix_ac <- function(prep, obs = NULL, latent = FALSE) {
   }
   # prepare residual standard deviations
   if (latent) {
-    sigma2 <- as.numeric(prep$ac$sderr)^2
+    if (!is.null(prep$ac$sderr)) {
+      sigma2 <- as.numeric(prep$ac$sderr)^2
+    }
+    if (!is.null(prep$ac$sdacranef)) {
```

Comment:
is "sdacranef" still needed? It seems to come from an old version of the code.

Reply:
ah, I think you're right; that was from the earliest version. will remove this.

Reply:
Thanks!